ControlMeasurements/Models/T.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ControlMeasurements.Models
+{
+    internal class T
```

Comment:
To raczej nie powinno tu byc :) "T" oznacza po prostu typ, nie tworz dla niego klasy

Reply:
Usunięte :)